documentation/compose/subject-strategy-demo/init/produce-messages.sh

```
@@ -0,0 +1,277 @@
+#!/bin/bash
+set -e
+
+echo "Waiting for Schema Registry to be ready..."
+until curl -s $SCHEMA_REGISTRY_URL/subjects > /dev/null 2>&1; do
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)